python/seldon_core/microservice.py

```diff
@@ -188,6 +195,11 @@ def setup_logger(log_level: str) -> logging.Logger:
     flask_logger = logging.getLogger("werkzeug")
     flask_logger.setLevel(log_level_num)
 
+    if getenv_as_bool(FILTER_METRICS_ACCESS_LOGS_ENV_NAME, default=False):
```

Comment:
Even though it would be a "breaking change", would it be too crazy to filter out the metrics logs by default? What are your thoughts on this @RafalSkolasinski ?

Reply:
Definitely something we could discuss but you're right on point - I wanted to avoid breaking changes

Reply:
I think it would make sense to set it as the default on `1.3`. We can always make it very clear on the release notes.

Reply:
It would also make sense to tie it to `debug` mode

Reply:
As in: if debug mode run verbose

Reply:
Cool, will make it default and will ping you when ready for re-review!

Reply:
@adriangonz made filtering default in production mode but disabled if `SELDON_DEBUG` flag is set, tell me what you think :)